hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -298,18 +298,21 @@ closeTx ::
   VerificationKey PaymentKey ->
   -- | The snapshot to close with, can be either initial or confirmed one.
   ClosingSnapshot ->
+  -- | Lower bound slot for the tx
+  SlotNo ->
   -- | Current slot and UTC time to compute the contestation deadline time.
```

Comment:
I think this comment is wrong. Could you maybe update it? It's not "current" because the other slot should be the "current" slot.